x/ecocredit/params.go

```diff
@@ -21,6 +23,8 @@ func ParamKeyTable() paramtypes.KeyTable {
 func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
 	return paramtypes.ParamSetPairs{
 		paramtypes.NewParamSetPair(KeyCreditClassFee, &p.CreditClassFee, validateCreditClassFee),
+		paramtypes.NewParamSetPair(KeyAllowlistedCreditCreators, &p.AllowedClassDesignerAddresses, validateAllowlistCreditDesigners),
```

Comment:
should we be consistent between the key naming and the field name? ie having `KeyAllowedClassDesignerAddresses` instead of `KeyAllowlistedCreditCreators` (even if it's a bit longer)
Maybe we could actually get rid of "addresses" at the end.
Same comment regarding the `validate...` function name, let's be consistent

Reply:
yeah, looks a bit better without "addresses" there. i updated the names here: https://github.com/regen-network/regen-ledger/pull/426/commits/da99e35fb880f9b957ba6caeca5148bee5c5ad70